wp-content/themes/reconasia/assets/_scss/components/_post-meta.scss

```
@@ -1,52 +1,68 @@
 @use 'abstracts' as *;
 
 .post-meta {
-  color: var(--post-meta);
-
   &__label {
     font-style: italic;
   }
 
   &__top {
     display: flex;
-    // @extend %font-label-lg;
   }
 
-  &__issue .post-meta__value,
   &__date {
-    letter-spacing: 1px;
-    text-transform: uppercase;
+    color: $color-black-165;
+    @extend %text-style-label-small-strong-caps;
   }
 
-  &__issue {
-    .post-meta__label {
-      text-transform: lowercase;
-    }
+  &__categories {
+    color: $color-link-primary-200;
+    @extend %text-style-label-x-small-strong-caps;
 
-    .post-meta__value {
-      font-weight: 600;
+    a:not(:last-of-type) {  
+      &::after {
+        content: '';
+        margin-right: rem(10);
+        margin-left: rem(10);
+        border-right: 1px solid $color-border-dark-130;
+      }
     }
-  }
 
-  &__date:not(:only-child) {
-    margin-left: rem(8);
+    > a {
+      &:hover {
+        color: $color-bg-dark-100;
+        transition: color 0.3s ease-in-out;
+      }
 
-    &::before {
-      content: '|';
-      margin-right: rem(8);
-      color: var(--post-meta-divider);
+      &:focus {
+        color: $color-bg-dark-100;
+        background: $color-bg-light-300;
+      }
     }
   }
 
   &__authors {
-    // @extend %font-body-sm;
-  }
+    margin-bottom: rem(4);
+    color: $color-black-190;
+    @extend %text-style-paragraph-small-short;
+
+    > a {
```

```suggestion
    a {
```

Same thing, the direct child selector isn't necessary.